x-pack/plugins/fleet/public/applications/integrations/sections/epm/screens/detail/assets/deferred_assets_accordion.tsx

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import type { FunctionComponent } from 'react';
+
+import { EuiSpacer, EuiCallOut, EuiTitle } from '@elastic/eui';
+
+import { i18n } from '@kbn/i18n';
+
+import { FormattedMessage } from '@kbn/i18n-react';
+
+import type { EsAssetReference } from '../../../../../../../../common';
+
+import type { PackageInfo } from '../../../../../types';
+import { ElasticsearchAssetType } from '../../../../../types';
+
+import { DeferredTransformAccordion } from './deferred_transforms_accordion';
+
+interface Props {
+  packageInfo: PackageInfo;
+  deferredInstallations: EsAssetReference[];
+}
+
+export const DeferredAssetsSection: FunctionComponent<Props> = ({
+  deferredInstallations,
+  packageInfo,
+}) => {
+  const deferredTransforms = deferredInstallations.filter(
+    (asset) => asset.type === ElasticsearchAssetType.transform
+  );
+  return (
+    <>
+      <EuiTitle>
+        <h2>
+          <FormattedMessage
+            id="xpack.fleet.epm.packageDetails.assets.deferredInstallationsLabel"
+            defaultMessage="Deferred installations"
+          />
+        </h2>
+      </EuiTitle>
+      <EuiSpacer size="l" />
+      <EuiCallOut
+        size="m"
+        color="warning"
+        iconType="alert"
+        title={i18n.translate(
+          'xpack.fleet.epm.packageDetails.assets.deferredInstallationsCallout',
+          {
+            defaultMessage:
+              'This package has {numOfDeferredInstallations, plural, one {one deferred installation} other {# deferred installations}} which might require additional permissions to install and operate correctly.',
```

Comment:
If action is definitely required and lack of permissions is definitely the cause of the problem, I think we should remove "might":
```suggestion
              'This package has {numOfDeferredInstallations, plural, one {one deferred installation} other {# deferred installations}} which require additional permissions to install and operate correctly.',
```

Reply:
Updated here [`98a01d4` (#154665)](https://github.com/elastic/kibana/pull/154665/commits/98a01d409be57bc46df59fcfbe6e17c92b71c780)